configure

```diff
@@ -883,17 +883,28 @@ case "${ARCH}" in
                     ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} adler32_neon.lo"
                 fi
             ;;
-            armv8-a+crc | armv8-a+crc+simd | armv8.[12]-a)
+            armv8-a | armv8-a+simd)
+                CFLAGS="${CFLAGS} -DUNALIGNED_OK -DUNROLL_LESS"
+                SFLAGS="${SFLAGS} -DUNALIGNED_OK -DUNROLL_LESS"
+
+                if test $buildneon -eq 1; then
+                    CFLAGS="${CFLAGS} ${floatabi} -DARM_NEON_ADLER32"
+                    SFLAGS="${SFLAGS} ${floatabi} -DARM_NEON_ADLER32"
+
+                    ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} adler32_neon.o"
+                    ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} adler32_neon.lo"
+                fi
+            ;;
+            armv8-a+crc | armv8-a+crc+simd | armv8.[1234]-a | armv8.[1234]-a+simd)
                 CFLAGS="-march=${ARCH} ${CFLAGS} -DARM_ACLE_CRC_HASH -DUNALIGNED_OK -DUNROLL_LESS"
                 SFLAGS="-march=${ARCH} ${SFLAGS} -DARM_ACLE_CRC_HASH -DUNALIGNED_OK -DUNROLL_LESS"
 
                 ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} crc32_acle.o insert_string_acle.o"
                 ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} crc32_acle.lo insert_string_acle.lo"
 
                 if test $buildneon -eq 1; then
-                    CFLAGS="${CFLAGS} ${floatabi} -mfpu=neon -DARM_NEON_ADLER32"
-                    SFLAGS="${SFLAGS} ${floatabi} -mfpu=neon -DARM_NEON_ADLER32"
-
+                    CFLAGS="${CFLAGS} ${floatabi} -DARM_NEON_ADLER32"
+                    SFLAGS="${SFLAGS} ${floatabi} -DARM_NEON_ADLER32"
```

Comment:
What's the reason for removing "-mfpu=neon"?

Reply:
On armv7, it requires "-mfpu=neon" to enable NEON instructions . But for armv8 , there's no need to use "-mfpu= neon", NEON instructions is enabled by default. And if compiled with "-mfpu=neon", there will be compiling errors.

Reply:
If it compiles correctly on @Dead2's Raspberry, then it's OK for me...

Reply:
I test it  on AArch64 servers. Does Raspberry  use armv8 or armv7 ?

Reply:
I think it has v8-A, but it only did boot 32-bit Linux.

Reply:
What's the output of "uname -m" on Raspberry